src/linter.js

```diff
@@ -28,14 +29,19 @@ export default async function linter(options, compiler) {
   let rawResults = [];
 
   try {
+    if (files.length === 0) {
+      throw new Error('No files matching');
```

Comment:
Not sure it should be the error, when nothing to lint we should do nothing

Reply:
I was just trying to make this pass existing tests. I agree, its odd. I'll remove it.

Reply:
Feel free to correct tests too